### PR TITLE
QOL-7939 merge theme into this extension

### PR DESCRIPTION
**Pull request by ThrawnCA**
This will simplify the CKAN 2.9 upgrade since the extension and theme are highly interdependent.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -75,10 +74,11 @@
     # pip to create the appropriate form of executable for the target platform.
     entry_points='''
         [ckan.plugins]
-        data_qld_resources=ckanext.data_qld.resources_plugin:DataQldResourcesPlugin
-        data_qld_integration=ckanext.data_qld.integration_plugin:DataQldIntegrationPlugin
+        data_qld=ckanext.data_qld.plugin:DataQldPlugin
+        data_qld_integration=ckanext.data_qld.plugin:DataQldPlugin
```
Comment (duttonw) [inline setup.py:78]: is it ok to load our plugin multiple times?

File: ckanext/data_qld/plugin.py [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        m.connect('dashboard.reports', '/dashboard/reporting', controller=controller, action='index')
+        m.connect(
+            '/dashboard/reporting/datasets/{org_id}/{metric}', controller=controller, action='datasets')
+        m.connect(
+            '/dashboard/reporting/datarequests/{org_id}/{metric}', controller=controller, action='datarequests')
+
+        return m
+
+    # IQA
+    def custom_resource_score(self, resource, resource_score):
```
Comment (duttonw) [inline ckanext/data_qld/plugin.py:225]: can't this custom_resource_score be in its own file like how before_show is called etc

File: setup.py (hunk 1 above)
Comment (ThrawnCA) [reply setup.py:78]: Not sure. I'm testing that now.

File: setup.py [hunk 3]
```diff
@@ -5,22 +5,22 @@
...
+    description='Custom extension for Data QLD',
+    long_description='',
 
     # The project's main homepage.
     url='https://github.com/qld-gov-au/ckanext-data-qld',
 
     # Author details
-    author='''Salsa Digital''',
-    author_email='''''',
+    author='Salsa Digital',
```
Comment (duttonw) [inline setup.py:22]: might be time to co-author this.

**Review by duttonw — APPROVED** [review]

File: ckanext/data_qld/plugin.py (hunk 2 above)
Comment (ThrawnCA) [reply ckanext/data_qld/plugin.py:225]: It could, but there is currently no associated submodule.

Comment (duttonw) [reply ckanext/data_qld/plugin.py:225]: was just meaning creating a new class for the function code to live in, so the plugin file is nice an clean.

File: setup.py (hunk 1 above)
Comment (ThrawnCA) [reply setup.py:78]: Looks like CKAN doesn't mind.

**Review by duttonw — APPROVED** [review]

**Pull request merged**